pwndbg/commands/telescope.py

```diff
@@ -92,14 +93,15 @@ def telescope(address=None, count=telescope_lines, to_string=False):
             break
 
         # Collapse repeating values.
-        value = pwndbg.memory.pvoid(addr)
-        if last == value:
-            if not skip:
-                result.append(T.repeating_marker('%s' % repeating_maker))
-                skip = True
-            continue
-        last = value
-        skip = False
+        if skip_repeating_values:
+            value = pwndbg.memory.pvoid(addr)
+            if last == value:
```

Comment:
This diff could be smaller if you just modify this to say `skip_repeating_values and ...`

Reply:
So you want it to be `if skip_repeating_values and last == value:`, right?
this would make us having an unnecessary read of `addr`, is that okay?